src/uploaders/ReactNativeUploader.ts

```diff
@@ -66,11 +69,39 @@ class ReactNativeUploader {
 
   private async getSourceMapAndBundle(
     projectRoot: string,
-    retrieval: SourceMapRetrieval
+    retrieval: SourceMapRetrieval,
+    platform: Platform,
+    dev: boolean
   ): Promise<SourceMapBundlePair> {
     switch (retrieval.type) {
-      case SourceMapRetrievalType.Fetch:
-        throw new Error('SourceMapRetrievalType.Fetch is not implemented!')
+      case SourceMapRetrievalType.Fetch: {
+        const queryString = qs.stringify({ platform, dev })
+        const sourceMapUrl = `${retrieval.url}/index.js.map?${queryString}`
+        const bundleUrl = `${retrieval.url}/index.bundle?${queryString}`
```

Comment:
I think we need to add an option for the entry point as it's customiseable, though I can't find instructions to do that on RN docs so maybe it's unnecessary?

Reply:
I think it will just bundle whatever is in the path of the URL? So yeah maybe that does need an option. Perhaps we need a set of `--bundler-*` options

- `--budler-url`
- `--bundler-entrypoint`
- `--bundler-` ?